packages/bits/src/lib/menu/menu-popup/menu-popup.component.ts

```diff
@@ -27,6 +27,7 @@ import { IMenuGroup, IMenuItem } from "../public-api";
     selector: "nui-menu-popup",
     host: {
         "class": "nui-menu-popup",
+        "role": "dialog",
```

Comment:
it seems like `menu` might be more appropriate here. `dialog` is a little more generic and doesn't tell the user exactly what to expect from this component. what do you think?

Reply:
Agree. Will changed, thanks